op-node/node/node.go

```diff
@@ -73,6 +76,10 @@ type OpNode struct {
 	// cancels execution prematurely, e.g. to halt. This may be nil.
 	cancel context.CancelCauseFunc
 	halted atomic.Bool
+
+	// conductor
+	conductorClient      conductorRpc.API
+	checkLeaderSequencer driver.CheckLeaderSequencerFunc
```

Comment:
Suggest to change this to a interface instead of a function. Since we'll have more than one interaction from sequencer to conductor:
1. commit unsafe payload to conductor
2. check is leader before publishing to p2p

Reply:
Please refer to https://github.com/ethereum-optimism/optimism/blob/develop/op-conductor/rpc/api.go#L15-L41 for API definition.